util/regexp-assemble/data/932105.data

```diff
@@ -131,69 +144,82 @@ nroff
 nsenter
 nslookup
 nstat
+octave@
+od
 onintr
 openssl
+openvpn
```

Comment:
FP? Maybe add `@`.

Reply:
On openvpn?

Reply:
Not sure. It's just that it's probably quite a common term to find in web documents. But then so are many of the other words in the list.

Reply:
Well, we can talk about this in today's meeting. Do we add `@` to any command that requires parameters?